src/doc/release/v1_10_0.md

```diff
@@ -0,0 +1,16 @@
+iCub 1.10.0 (UNRELEASED)  Release Notes
+========================
+
+New Features
+------------
+
+### Libraries
+
+#### `skinDynLib`
+Add `linkName` and `frameName` attributes to `iCub::skinDynLib::dynContact` and `iCub::skinDynLib::skinContact`, to simplify 
```

Comment:
Am I wrong or going to a newline in markdown has no effect? I think that writing paragraphs on a single line is the best choice, and new line behavior should be handled by the soft wrap configuration of the editor the user uses.